package.json

```diff
@@ -157,6 +157,7 @@
   },
   "dependencies": {
     "@parity/api": "2.1.20",
+    "@parity/dapp-wallet": "github:js-dist-paritytech/dapp-wallet#037638fec1d3146bc3d3980597cee62b37617373",
```

Comment:
Probably this should go in devDependencies for consistency.

Reply:
yes, totally